packaging/hudi-aws-bundle/pom.xml

```diff
@@ -284,19 +275,13 @@
             <artifactId>hudi-aws</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <!-- Need parquet and avro to run AwsGlueCatalogSyncTool using run_sync_tool with this bundle.
-        Parquet and avro from other packages have already been shaded above-->
+
+        <!-- Parquet -->
         <dependency>
             <groupId>org.apache.parquet</groupId>
             <artifactId>parquet-avro</artifactId>
             <version>${parquet.version}</version>
             <scope>compile</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.avro</groupId>
```

Comment:
Shouldn't we do the same for GCP bundle?

Reply:
You're right. The same reasoning should apply to gcp bundle as well. HUDI-4889 to track.
I am targeting it for 0.13.0 for the lack of time to test the changes right now.